docs/store.js

```diff
@@ -329,23 +330,22 @@ var LocalCalendarStorage = function(params) {
         
         //store the state in localStorage
         var stateP = new Promise(function(resolve, reject) {
-            storeInLocalStorage(toKey(calendarObj.state.uniqueId), calendarObj.state);
+            localStorage.setItem(toKey(calendarObj.state.uniqueId), JSON.stringify(calendarObj.state));
```

Comment:
Since you have to remember to do a few different things for this to work correctly, (`toKey`... `stringify`...) making a little internal helper function would have been better here. Then you don't have as much copy paste noise all over the place, but the local storage is still kept internal to the store.

```
self.set = function(key, val) {
   localstorage.setItem(toKey(key), JSON.stringify(val));
}
```